examples/classifcation/text_classification.exs

```diff
@@ -0,0 +1,273 @@
+# Based on https://valueml.com/named-entity-recognition-using-lstm-in-keras/
+
+# References
+# http://alexminnaar.com/2019/08/22/ner-rnns-tensorflow.html
+
+# Test Data: https://github.com/bhuvanakundumani/NER_tensorflow2.2.0/blob/master/data/test.txt
+
+# Train Data: https://github.com/bhuvanakundumani/NER_tensorflow2.2.0/blob/master/data/train.txt
+
+# Validation Data: https://github.com/bhuvanakundumani/NER_tensorflow2.2.0/blob/master/data/valid.txt
+
+Mix.install([
+  {:axon, github: "elixir-nx/axon"},
+  {:nx, "~> 0.2.1"},
+  {:exla, "~> 0.2.2"}
+])
+
+EXLA.set_as_nx_default([:tpu, :cuda, :rocm, :host])
+
+defmodule TextClassification do
+  require Axon
+  # From Example http://alexminnaar.com/2019/08/22/ner-rnns-tensorflow.html
+  EXLA.set_as_nx_default([:tpu, :cuda, :rocl, :host]) # My :host option doesnt work, jit_apply is undefined
+  require Logger
+  @sequence_length 75
+  @batch_size 128
+  @lstm 128
+  @embed_dimension 256
+  @def_split " "
+  @start "-DOCSTART- -X- -X- O\n"
+  @ending "-------------------------------------------------------------"
+  @incorrect [".", "?", "%", ",", ":", ";", "-docstart-"]
+
+  # seqlen
+  # word_count
+
+  def train(model, data) do
+    model
+    |> Axon.Loop.trainer(&kl_divergence/2, Axon.Optimizers.adam(0.001))
+    |> Axon.Loop.metric(:accuracy, "Accuracy")
+    |> Axon.Loop.run(data, %{}, epochs: 3, iterations: 5)
+
+  end
+
+  defp kl_divergence(y_true, y_pred) do
+    Axon.Losses.kl_divergence(y_true, y_pred, reduction: :mean)
+  end
+
+  def build_model(word_count, label_count) do
+      Axon.input({nil, @sequence_length, 1}, "inputs")
+       |> Axon.embedding(word_count, @sequence_length)
+       |> Axon.nx(fn t ->
```

Comment:
The reason you have sized 1 is because your input shape is `{nil, seq_len, 1}`. My suggestion is to squeeze your inputs such that the new input shape is `{nil, seq_len}` instead, and then you can remove this